src/web_interface/filter.py

```diff
@@ -366,3 +379,16 @@ def is_not_mandatory_analysis_entry(item: str, additional_entries: Optional[List
         item not in ['analysis_date', 'plugin_version', 'skipped', 'summary', 'system_version', 'tags']
         and (additional_entries is None or item not in additional_entries)
     )
+
+
+def random_collapse_id():
+    return ''.join((random.choice(ascii_letters) for _ in range(10)))
+
+
+def version_links(versions, selected_analysis=None):
+    if selected_analysis:
+        template = '<a href="/analysis/{{}}/{}">{{}}</a>'.format(selected_analysis)
+    else:
+        template = '<a href="/analysis/{}">{}</a>'
+
+    return [template.format(version['_id'], version['version']) for version in versions]
```

Comment:
it seems to me `version` is not a particularly good name for the variable here. maybe `other_fw` or something idk